java-engine/src/main/java/io/getunleash/engine/UnleashEngine.java

```diff
@@ -49,7 +49,7 @@ public UnleashEngine(
       IStrategy fallbackStrategy,
       NativeInterface nativeInterface) {
     if (customStrategies != null && !customStrategies.isEmpty()) {
-      List<String> builtInStrategies = new ArrayList<>();
+      List<String> builtInStrategies = getBuiltInStrategies();
```

Comment:
So this evaluates all custom and built in strategies (as it's Java bytecode now)